scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -0,0 +1,346 @@
+# Copyright 2020 Google LLC
```

Comment:
Can you add a test? A few rows from BOLTS original data, and the expected output of this script?